### DISCUSSION TITLE
How I can catch all events that failed to be written in sink (Elasticearch)?

### DISCUSSION THREAD

**Discussion by gl1ridae** [vectordotdev/vector#21184] [Q&A] [answered]
Hi all! I have pretty big flow of events which written into different Elasticsearch (ES) indices based on some fields. And it happens that events cannot be written because of schema errors or mapping exceptions from ES side. The question is what is the conventional way to catch these events, which failed to be written into ES, and write it into file sink for example?

**Comment by jszwedko** [comment, marked as answer]
Hey! Unfortunately you can't yet. This feature is being tracked by https://github.com/vectordotdev/vector/issues/1772

**Comment by gl1ridae** [comment]
Got it, thanks!
